source/cpu/cpu.go

```diff
@@ -173,8 +174,11 @@ func (s *cpuSource) GetLabels() (source.FeatureLabels, error) {
 	}
 
 	// Security
+	skipLabel := sets.NewString("tdx.total_keys")
```

Comment:
Maybe add a comment?
```suggestion
        // A set of features that will not have labels created but are only made available for
        // NodeFeatureRules (e.g. to be published via extended resources instead)
	skipLabel := sets.NewString("tdx.total_keys")
```

Reply:
👍 Added a commet